packages/slate/src/models/node.js

```diff
@@ -1109,6 +1109,11 @@ class Node {
 
     const text = this.getDescendant(range.startKey)
     const char = text.characters.get(range.startOffset)
+    // PREF: if text.characters.size === 0; then no char is got
+    // empty text by normalization or delete at range
```

Comment:
Hey @zhujinxuan can you remove this comment? Usually `PERF:` comments should be things that could be removed without any breakage and are only for performance improvements. In this case that doesn't apply.